client/flutter/lib/home_page.dart

```diff
@@ -6,11 +7,22 @@ import 'package:WHOFlutter/pages/travel_advice.dart';
 import 'package:WHOFlutter/pages/who_myth_busters.dart';
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
 import 'package:share/share.dart';
 
 class HomePage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
+    SystemChrome.setSystemUIOverlayStyle(SystemUiOverlayStyle(
+      statusBarColor: Colors.transparent,
+      statusBarIconBrightness: Brightness.dark,
+      statusBarBrightness:
+          Platform.isAndroid ? Brightness.dark : Brightness.light,
```

Comment:
Did you test this on Web? AFAIK `Platform.isAndroid` returns `null` on Web, open Flutter issue: https://github.com/flutter/flutter/issues/36126

Reply:
no, I was on flutter stable channel,

Reply:
its been tested on web and android.

Reply:
`statusBarBrightness` is only respected on IOS so that check isn't required. I still think that these changes are relevant after some tweaks if design is happy with it. cc @hspinks